pkgs/create-neon/data/templates/ts/index.mts.hbs

```diff
@@ -0,0 +1 @@
+export { hello, Greeting } from './index.cjs';
```

Comment:
It really is unfortunate. I wish we could do `export * from './index.cjs';` but when I tried this it broke in confusing ways. I think it had something to do with the fact that the CJS exports are computed dynamically. Open to ideas, though.

Reply:
You know, I just tested it and it seems to work. It might've been an issue only when integrating with the `ncc` bundler, but I don't want to make everyone pay this tax just for users of bundlers. Let's go with `export * from` by default, and then only people using bundlers will have to deal with this unfortunate issue.

Reply:
Proof: https://github.com/dherman/neon-rs/pull/47